tests/core/test_call_stop.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+from __future__ import print_function
+
+import pytest
+
+from circuits import handler, Component, Event
+
+
+class test(Event):
+    """test Event"""
+
+    success = True
+
+
+class foo(Event):
+    """foo Event"""
+
+
+class App(Component):
+
+    @handler("test", priority=1)
+    def on_test(self, event):
+        event.stop()
+        yield
+
+    @handler("test")
+    def on_test_ignored(self):
+        return "Hello World!"
+
+
+@pytest.fixture
+def app(request, manager, watcher):
+    app = App().register(manager)
+    assert watcher.wait("registered")
+
+    def finalizer():
+        app.unregister()
+
+    request.addfinalizer(finalizer)
+
+    return app
+
+
+def test_call_stop(manager, watcher, app):
```

Comment:
Do we need to get this fancy here (ie with `manager` in a new thread etc).
Wouldn't this work too:
```
def test_call_stop():
  app = App()
  x = app.fire(test())
  app.tick()
  
  assert x.success # Or whatever is needed to check that the event ran properly.
  assert.x.value is None
```

This way we do not run into the overhead of creating new threads and bussy waiting for events which makes tests slower :D